R/uvhydrograph-render.R

```diff
@@ -180,47 +180,44 @@ createPrimaryPlot <- function(
 
   #Don't add the right-side axis if we aren't actually plotting anything onto it
   if((limsAndSides[['sides']][['reference']] == 4) || (limsAndSides[['sides']][['comparison']] == 4)){
-    plot_object <- lines(plot_object, x=0, y=0, side = 4, reverse = primarySeriesList[['inverted']]) %>%
+     plot_object <- lines(plot_object, x=0, y=0, side = 4, reverse = primarySeriesList[['inverted']]) %>%
         axis(side = 4, las = 0, reverse = primarySeriesList[['inverted']])
   }
   
+  #uncorrected data
+  if(!isEmptyOrBlank(primarySeriesList[['uncorrected']]) && !isEmptyVar(primarySeriesList[['uncorrected']][['points']])) {
+    plot_object <- plotTimeSeries(plot_object, primarySeriesList[['uncorrected']], "uncorrected", 
+                                  timezone, getPrimaryPlotConfig, list(uvInfo[['label']], limsAndSides$ylims[['primary']], limsAndSides$sides[['primary']]), excludeZeroNegativeFlag)
```

Comment:
confused about how `doLog` ever makes it to `getPrimaryPlotConfig`. It should be in the list, but I see labels, limits, and sides added. Is it stored in `limsAndSides$sides[['primary']]`?